app/instrumentation-tests/src/org/commcare/androidTests/OtherActionTests.kt

```diff
@@ -0,0 +1,266 @@
+package org.commcare.androidTests
+
+
+import android.view.KeyCharacterMap
+import android.view.KeyEvent
+import android.view.View
+import android.view.ViewGroup
+import android.view.inputmethod.EditorInfo
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import junit.framework.Assert.assertTrue
+import org.commcare.annotations.BrowserstackTests
+import org.commcare.dalvik.R
+import org.commcare.utils.CustomMatchers
+import org.commcare.utils.InstrumentationUtility
+import org.commcare.utils.isPresent
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers
+import org.hamcrest.Matchers.*
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.Description
+import org.junit.runner.RunWith
+import java.util.*
+
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+@BrowserstackTests
+class OtherActionTests: BaseTest() {
+    companion object {
+        const val CCZ_NAME = "other.ccz"
+        const val APP_NAME = "Other Test"
+    }
+
+    @Before
+    fun setup() {
+        installApp(APP_NAME, CCZ_NAME)
+        InstrumentationUtility.login("test1", "123")
+    }
+
+    @Test
+    fun testActionKey() {
```

Comment:
This test as it is not very meaningful/complete today as it's meant to check that ime actions are working correctly, but we are simply performing the ime actions without checking the result of performing the ime action.  I recognise it might not be possible to check for the result of ime actions, so we should probably remove this test from the PR and keep it manual for now.